pkg/api/v1alpha1/tinkerbelltemplateconfig_defaults.go

```diff
@@ -29,39 +31,39 @@ warnings:
 `
 )
 
-func GetDefaultActionsFromBundle(b v1alpha1.VersionsBundle) []ActionOpt {
+func GetDefaultActionsFromBundle(b v1alpha1.VersionsBundle, diskType string) []ActionOpt {
 	return []ActionOpt{
-		withStreamImageAction(b),
-		withNetplanAction(b),
-		withTinkCloudInitAction(b),
-		withDsCloudInitAction(b),
-		withKexecAction(b),
+		withStreamImageAction(b, diskType),
+		withNetplanAction(b, diskType),
+		withTinkCloudInitAction(b, diskType),
+		withDsCloudInitAction(b, diskType),
+		withKexecAction(b, diskType),
 	}
 }
 
-func withStreamImageAction(b v1alpha1.VersionsBundle) ActionOpt {
+func withStreamImageAction(b v1alpha1.VersionsBundle, diskType string) ActionOpt {
 	return func(a *[]tinkerbell.Action) {
 		*a = append(*a, tinkerbell.Action{
 			Name:    "stream-image",
 			Image:   b.Tinkerbell.TinkerbellStack.Actions.ImageToDisk.URI,
 			Timeout: 360,
 			Environment: map[string]string{
 				"IMG_URL":    b.EksD.Raw.Ubuntu.URI,
-				"DEST_DISK":  "/dev/sda",
+				"DEST_DISK":  diskType,
 				"COMPRESSED": "true",
 			},
 		})
 	}
 }
 
-func withNetplanAction(b v1alpha1.VersionsBundle) ActionOpt {
+func withNetplanAction(b v1alpha1.VersionsBundle, diskType string) ActionOpt {
 	return func(a *[]tinkerbell.Action) {
 		*a = append(*a, tinkerbell.Action{
 			Name:    "write-netplan",
 			Image:   b.Tinkerbell.TinkerbellStack.Actions.WriteFile.URI,
 			Timeout: 90,
 			Environment: map[string]string{
-				"DEST_DISK": "/dev/sda2",
+				"DEST_DISK": fmt.Sprintf("%s2", diskType),
```

Comment:
We are just appending `2` to `diskType`. Do we account for whether `diskType` is `sda` or `nvme`?
Because for `sda` you can just append 2 but with nvme you need `p2`

Reply:
We talked about dealing with NVMe on a separate PR so we can unblock mainline.

Reply:
Yes, I am going to create a separate PR for this.